### DISCUSSION TITLE
Clarification: Is creating a Sub-task and assigning a Parent-task are the same?

### DISCUSSION THREAD

**Discussion by madukan** [makeplane/plane#5482] [General]
Let me clarify through two scenarios. Are these two scenarios the same? In the sense plane.so would result in queries in the same way in different views?

#### Scenario 1:
 I create an issue (task), and the create a sub-task under that view. Say now task A has sub-task B.
#### Scenario 2: 
In another instance, I create issue (task) X and the create another issue (task) Y. Then I set parent task in Y, as X.

#### My question:
Is A and B has the same structure as X and Y?
In the sense, the queries/views return same results in same for both of these relationships?

PS: I just got started with the plane.so and must it it rocks!

**Comment by madukan** [comment]
Internally (code) I haven't checked yet (some yarn errors at the moment), but externally it appears to be. So it I assume (based on visuals) it is a `yes`. Correct me please if I am wrong.

**Comment by BraianS** [comment]
Any news on this?
